config/src/wsv.rs

```diff
@@ -0,0 +1,42 @@
+//! Module for `WorldStateView`-related configuration and structs.
+
+use iroha_config_base::derive::Configurable;
+use iroha_data_model::{metadata::Limits as MetadataLimits, LengthLimits};
+use serde::{Deserialize, Serialize};
+
+use crate::wasm;
+
+const DEFAULT_METADATA_LIMITS: MetadataLimits = MetadataLimits::new(2_u32.pow(20), 2_u32.pow(12));
+const DEFAULT_IDENT_LENGTH_LIMITS: LengthLimits = LengthLimits::new(1, 2_u32.pow(7));
+
+/// `WorldStateView` configuration.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Deserialize, Serialize, Configurable)]
+#[config(env_prefix = "WSV_")]
+#[serde(rename_all = "UPPERCASE", default)]
+pub struct Configuration {
+    /// [`MetadataLimits`] for every asset with store.
+    pub asset_metadata_limits: MetadataLimits,
+    /// [`MetadataLimits`] of any asset definition metadata.
+    pub asset_definition_metadata_limits: MetadataLimits,
+    /// [`MetadataLimits`] of any account metadata.
+    pub account_metadata_limits: MetadataLimits,
+    /// [`MetadataLimits`] of any domain metadata.
+    pub domain_metadata_limits: MetadataLimits,
```

Comment:
What about

```rust
struct MetadataLimits<T> {
    // ...
    _phantom: PhantomData<T>,
}
```

Not for this PR, just an idea for a future

Reply:
Strong typing is always great, we should consider this refactoring

Reply:
Created #2509